gradoop-hbase/src/test/resources/data/gdl/social_network.gdl

```diff
@@ -0,0 +1,44 @@
+// all vertices
```

Comment:
I guess these test resources should stay in gradoop-common. I'm not sure, but are the HBase tests the only consumers of those?